kube/manifests/thanos-store.yaml

```diff
@@ -39,6 +39,10 @@ spec:
           containerPort: 10901
         - name: cluster
           containerPort: 10900
+        livenessProbe:
```

Comment:
This is quite wrong, as this will fail for most of store gateway failures for bigger buckets.

It takes time for store to be ready and there is no good solution to that. By specifying liveness probe (with unknown timeout, retries options) we risk that kubernetes will always restart our container because it thinks pod is stuck.

This could be readiness probe though.